test/appframework_aws/c3/appframework_aws_test.go

```diff
@@ -919,17 +940,31 @@ var _ = Describe("c3appfw test", func() {
 				},
 			}
 
+			// Create App framework Spec for Indexer Cluster
+			appSourceNameIdxc = "appframework-idxc-" + enterpriseApi.ScopeCluster + testenv.RandomDNSName(3)
+			appSourceVolumeNameIdxc := "appframework-test-volume-idxc-" + testenv.RandomDNSName(3)
+			appFrameworkSpecIdxc := testenv.GenerateAppFrameworkSpec(ctx, testcaseEnvInst, appSourceVolumeNameIdxc, enterpriseApi.ScopeCluster, appSourceNameIdxc, s3TestDirIdxc, 60)
+
 			// Deploy C3 SVA
-			// // Deploy the Cluster Master
-			// testcaseEnvInst.Log.Info("Deploy Cluster Manager")
-			// _, err = deployment.DeployClusterMaster(ctx, deployment.GetName(), "", "", "")
-			// Expect(err).To(Succeed(), "Unable to deploy Cluster Manager")
+			// Deploy the Cluster Master
+			testcaseEnvInst.Log.Info("Deploy Cluster Master")
+			cmSpec := enterpriseApiV3.ClusterMasterSpec{
+				CommonSplunkSpec: enterpriseApi.CommonSplunkSpec{
+					Spec: enterpriseApi.Spec{
+						ImagePullPolicy: "Always",
+					},
+					Volumes: []corev1.Volume{},
+				},
+				AppFrameworkConfig: appFrameworkSpecIdxc,
+			}
+			_, err = deployment.DeployClusterMasterWithGivenSpec(ctx, deployment.GetName(), cmSpec)
```

Comment:
Returns the CM and can be used instead of the get Instance function called later